src/linear_solver_routines/block_row_projection.jl

```diff
@@ -0,0 +1,45 @@
+# This file is pat of RLinearAlgebra.jl
+"""
+    LinSysBlkRowLQ <: LinSysBlkRowProjection
+
+A mutable structure that represents a standard block row projection method.
+
+# Aliases
+- `BlockKaczmarz`
+
+# Fields
+- `alpha::Float64`, a relaxation parameter that should be set between `0.0` and `2.0`
+- `update::Union{Nothing, AbstractArray}` - A buffer for storing update.
+
+Calling `LinSysBlkRowProj()` defaults the relaxatoin parameter to `1.0`.
+"""
+mutable struct LinSysBlkRowLQ <: LinSysBlkRowProjection 
+    α::Float64
+    update::Union{Nothing, AbstractArray}
+end
+LinSysBlkRowLQ(α) = LinSysBlkRowLQ(α, nothing)
+
+LinSysBlkRowLQ() = LinSysBlkRowLQ(1.0, nothing)
+
+BlockKaczmarz = LinSysBlkRowLQ
+# Common rsubsolve interface for linear systems
+function rsubsolve!(
+    type::LinSysBlkRowLQ,
+    x::AbstractVector,
+    samp::Tuple{U,V,W} where {U<:Union{AbstractVector,AbstractMatrix},V<:AbstractArray,W<:AbstractVector},
+    iter::Int64,
+)
+    # samp[1] is the search direction
+    # samp[2] is the sketched matrix A
+    # samp[3] is the residual of system in A * samp[1], (samp[1] * A * x - b)
+    if iter == 1
+        # get the dimensions to allocate the update vector
+        p,n = size(samp[2])
+        type.update = Array{typeof(samp[2][1])}(undef, n)
+    end
+    # Reset the update vector 
+    fill!(type.update, 0)
```

Comment:
Figure out what is going on with `lq` and why this needs to be here

Reply:
Not entirely sure, but the `lq`  implementation of the `ldiv!`, seems to have a views usage that is problematic. Using factorize() instead of lq() seems to handle this appropiately.